app/views/admin/stolen_bikes/edit.html.haml

```diff
@@ -1,110 +1,135 @@
+    .form-group
+      = f.label :user_email
+      = f.email_field :user_email, value: @bike.owner_email, class: "form-control w-25"
+    .form-group
+      = f.label :bike, "Your stolen bike"
+      = f.text_field :title, value: "Your stolen #{@bike.title_string}", class: "form-control w-25"
+    = f.hidden_field :contact_type, value: 'stolen_contact'
+    = f.hidden_field :creator_id, value: current_user.id
+    = f.hidden_field :bike_id, value: @bike.id
+    .form-group
+      = f.label :notification_body
+      = f.text_area :body, placeholder: "You can write HTML in here. There will be a link to the #{@bike.type} included automatically.", rows: 7, class: "form-control w-25"
+    .row
+      .col-sm
+      .col-sm
+      .col-sm
+        = f.submit 'Send it!', class: 'btn btn-success'
+
+  %ul.bike-boxes
+    -# Should I create a new partial here?
+    = render partial: "/bikes/bike", locals: {bike: @bike, target: "show" }
+  - if @stolen_record.present?
+    %table.table-list
+      %tbody
+        %tr
+          %td
+            Theft description
+          %td
+            = @stolen_record.theft_description
+        %tr
+          %td
+            Created
+          %td
+            = @bike.created_at.strftime("%m-%d-%Y")
+        %tr
+          %td
+            Owner email
+          %td
+            = @bike.owner_email
+        %tr
+          %td
+            Current owner
+          %td
+            -# Unsure of what to put here
+            = @bike.owner_email
+        -if @bike.creation_organization_id.present?
+          %tr
+            %td
+              Organization name
+            %td
+              = @bike.creation_organization.name
+        %tr
+          %td
             Callable by
-          %dd
+          %td
             - if @stolen_record.phone_for_everyone
               %strong
                 Everyone
-            - else
+            -else
               = "Users" if @stolen_record.phone_for_users
-              = " Shops" if @stolen_record.phone_for_shops
-              = " Police" if @stolen_record.phone_for_police
-          .clearfix
-          %dt
+              = "Shops" if @stolen_record.phone_for_shops
+              = "Police" if @stolen_record.phone_for_police
+        %tr
+          %td
             Link to recover
-          %dd
+          %td
             %input{ disabled: true, type: 'text', value: edit_bike_recovery_url(bike_id: @bike.id, token: @stolen_record.find_or_create_recovery_link_token).to_s }
-          - if @stolen_record.approved
-            %h2
-              Stolen bike approved
-          - else
-            = link_to 'Approve listing', approve_admin_stolen_bike_url(@bike), method: :post, confirm: "Approving will:\n\n- Send out a tweet\n- The #{@bike.type} will be posted to any pertinent blogs.\n- It can NOT be effectively reversed\n", class: 'button-blue padded'
-        %dl.dl-horizontal
-        
-          %dt
+        %tr
```

Comment:
On large screens, I think this table should break into two tables, half screen width each